src/index.js

```diff
@@ -26,10 +27,18 @@ exports["default"] = () => {
             });
 
             this.logManager = new LogManager();
-            await this.logManager.addAppenders(
-                { type: ConsoleLogAppender },
-                { type: ReportPortalAppender }
-            );
+            const debugMode = ( cliArguments.rdebug === "true")? true:false;
+            if(debugMode){
+                await this.logManager.addAppenders(
+                    { type: ConsoleLogAppender }
```

Comment:
I think we would like it the other way, when we debug, we want mostly report portal debugs and without the tests printing to console